sdk/documenttranslation/Azure.AI.DocumentTranslation/src/Models/TranslationTarget.cs

```diff
@@ -11,34 +11,38 @@ namespace Azure.AI.DocumentTranslation.Models
     public partial class TranslationTarget
     {
         [CodeGenMember("StorageSource")]
-        internal string StorageSource { get; set;}
+        internal string StorageSource { get; set; }
 
         /// <summary> Location of the folder / container with your documents. </summary>
         [CodeGenMember("TargetUrl")]
-        public Uri TargetUrl { get; }
+        public Uri TargetUri { get; }
+
+        /// <summary> Target Language. </summary>
+        [CodeGenMember("Language")]
+        public string LanguageCode { get; }
 
         /// <summary> Initializes a new instance of TargetInput. </summary>
-        /// <param name="targetUrl"> Location of the folder / container with your documents. </param>
-        /// <param name="language"> Target Language. </param>
-        /// <param name="glossaries"></param>
-        /// <exception cref="ArgumentNullException"> <paramref name="targetUrl"/> or <paramref name="language"/> is null. </exception>
-        public TranslationTarget(Uri targetUrl, string language, IList<TranslationGlossary> glossaries)
+        /// <param name="targetUri"> Location of the folder / container with your documents. </param>
+        /// <param name="languageCode"> Target Language. </param>
+        /// <param name="glossaries"> List of <see cref="TranslationGlossary"/>. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="targetUri"/> or <paramref name="languageCode"/> is null. </exception>
+        public TranslationTarget(Uri targetUri, string languageCode, IList<TranslationGlossary> glossaries)
```

Comment:
Curious, did you add this method? or was it created by codegen?
I ask because `glossaries` is not required so shouldn't be in the constructor

Reply:
I added it. But we don't really need it we can just add glossaries to the list instead of initializing it and passing it to the constructor.